address.go

```diff
@@ -80,16 +84,16 @@ func (m *AddressMessage) UnmarshalBinary(b []byte) error {
 	m.Index = nativeEndian.Uint32(b[4:8])
 
 	if l > unix.SizeofIfAddrmsg {
-		m.Attributes = AddressAttributes{}
 		ad, err := netlink.NewAttributeDecoder(b[unix.SizeofIfAddrmsg:])
 		if err != nil {
 			return err
 		}
-		ad.ByteOrder = nativeEndian
```

Comment:
As noted in a few other repos, native endian is the default byte order anyway, hence deleting these lines.